dotcom-rendering/src/web/components/App.tsx

```diff
@@ -338,7 +337,7 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 			country: countryCode,
 			pubData: {
 				platform: 'next-gen',
-				browserId,
+				browserId: browserId ?? undefined,
```

Comment:
Ah, so here you're transforming back to `undefined` so we're certain there are no down stream impacts? Clever

Might be worth adding a comment on this so no-one comes along later and cleans this up to just pass down `browserId` - I can see how that could be tempting.

Reply:
In the future, this way of sending `pubData` will likely be deprecated. I’ll let @shtukas share his latest on the matter.

```suggestion
				browserId: browserId ?? undefined, // if `undefined`, the resulting consent signal cannot be joined to a page view.
```